BfA/Underrot/Trash.lua

```diff
@@ -130,6 +131,7 @@ function mod:OnBossEnable()
 	-- Fallen Deathspeaker
 	self:Log("SPELL_CAST_START", "RaiseDead", 272183)
 	self:Log("SPELL_CAST_START", "WickedFrenzy", 266209)
+	self:Log("SPELL_AURA_APPLIED", "WickedEmbraceApplied", 266265)
```

Comment:
Can you move this down 1 line to keep the Wicked Frenzy declarations together and match the ordering of the event handlers below?